kili/cli/project/member/list_.py

```diff
@@ -0,0 +1,76 @@
+"""CLI's project list subcommand"""
+
+from typing import Dict, List, Optional, cast
+import click
+
+import pandas as pd
+from tabulate import tabulate
+
+from kili.client import Kili
+from kili.cli.common_args import Options
+
+ROLE_ORDER = {
+    v: i for i, v in enumerate(["ADMIN", "TEAM_MANAGER", "REVIEWER", "LABELER"])
+}
+
+
+def role_order(col: pd.Series) -> pd.Series:
+    """ordering pandas series by custom role order"""
+    out = col
+    # apply custom sorting only to column one:
```

Comment:
putting comments is often a bad habit.
At Kili, we do not write comments. Object names should be enough to understand what you are trying to achieve
https://medium.com/@bpnorlander/stop-writing-code-comments-28fef5272752

Reply:
Sorry, it came from a copy-paste, I forgot to remove them.